.travis.yml

```diff
@@ -4,33 +4,51 @@
 language: generic
 
 os: osx
-osx_image: beta-xcode6.1
+osx_image: xcode6.4
 
 env:
   matrix:
     
     - CONDA_PY=27
-    - CONDA_PY=34
     - CONDA_PY=35
```

Comment:
hmm, not sure why, but smithy isn't setting this to py36, not immediately obvious to me why